tests/PostgreSQLQueryBuilderTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\Queries\SQLSelect;
+use SilverStripe\PostgreSQL\PostgreSQLQueryBuilder;
+
+class PostgreSQLQueryBuilderTest extends SapphireTest
+{
+    public function testLongAliases()
+    {
+        $query = new SQLSelect();
+        $longstring = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+        $alias2 = $longstring . $longstring;
+        $query->selectField('*');
+        $query->addFrom('"Base"');
+        $query->addLeftJoin(
+            'Joined',
+            "\"Base\".\"ID\" = \"{$alias2}\".\"ID\"",
+            $alias2
+        );
+        $query->addWhere([
+            "\"{$alias2}\".\"Title\" = ?" => 'Value',
+        ]);
+
+        $identifier = "c4afb43_hijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+        $this->assertEquals(PostgreSQLQueryBuilder::MAX_TABLE, strlen($identifier));
```

Comment:
I'm not entirely sure what this is testing, but arguments to `assertEquals` is `$expected, $actual` so perhaps these arguments should be flipped?